### DISCUSSION TITLE
How to bind edit events

### DISCUSSION THREAD

**Discussion by YangFong** [dream-num/univer#1486] [Q&A | 常见问题] [answered]
How to bind edit cell events, I did not find the relevant way.

**Comment by Dushusir** [comment, marked as answer]
> How to bind edit cell events, I did not find the relevant way.

You can use the Facade or Uniscript API to monitor command execution
```js
univerAPI.onCommandExecuted((command, options) => {
  if (command.id === 'sheet.command.set-range-values') {
    console.log(command)
  }
})
```
More tutorial https://univer.ai/guides/facade/
